docs/csharp/misc/cs0236.md

```diff
@@ -26,13 +44,47 @@ public class MyClass
 {
     public int i = 5;
 
-    // To fix the error, remove "= i", and uncomment the line in constructor.
+    // CS0236: Field initializer cannot reference instance field 'i'
+    // This fails because 'i' might not be initialized when 'j' is being initialized
```

Comment:
This isn't quite true. The rule exists to class authors can rearrange fields without introducing compiler errors.